includes/class-sensei-lesson.php

```diff
@@ -2557,23 +2561,38 @@ public function lesson_add_existing_questions() {
 
 				foreach ( $questions as $question_id ) {
 
-					++$question_count;
-
 					$quizzes = get_post_meta( $question_id, '_quiz_id', false );
+
 					if ( ! in_array( $quiz_id, $quizzes ) ) {
+
```

Comment:
Can we get rid of this blank line on line 2567?  Also on line 2577.

Reply:
Done!